tasks/RevolutionTask.cpp

```diff
@@ -29,6 +29,17 @@ void tryParseAndWriteIgnoringExceptions(function<void()> write_func)
     }
 }
 
+void RevolutionTask::configureGetRequest(Time const& update_interval, std::string request)
+{
+    GetRequestConfig request_config;
+    request_config.update_interval = update_interval;
+    if (!update_interval.isNull()) {
+        request_config.trigger_deadline = Time::now();
+        request_config.request = request;
+    }
+    m_get_requests.push_back(request_config);
+}
+
```

Comment:
If the update_interval is null, do you still have to "push_back" the request_config?